### PR TITLE
Tweak: english text improvements

### PR DESCRIPTION
**Pull request by gabeklavans**

### CODE REVIEW

File: App/Features/Setting/SettingView.swift [hunk 1]
```diff
@@ -18,17 +18,17 @@ struct SettingView: View {
                     }
                 }
             }
-            Section("Badge") {
-                Toggle("Show badge", isOn: $badge)
+            Section("Notifications") {
+                Toggle("Show unread articles badge", isOn: $badge)
```
Comment (bourvill) [inline App/Features/Setting/SettingView.swift:22]: It is not related to the unread articles, but rather related to the total articles of the user’s default setting.

File: App/Features/WallabagPlus/WallabagPlusView.swift [hunk 2]
```diff
@@ -5,17 +5,17 @@ struct WallabagPlusView: View {
...
         VStack(alignment: .leading) {
-            GroupBox("What is wallabag Plus ?") {
-                Text("Wallabag plus offert premium feature powered by AI")
+            GroupBox("What is wallabag Plus?") {
+                Text("wallabag Plus offers premium features powered by AI")
                     .frame(idealWidth: .infinity, maxWidth: .infinity, alignment: .leading)
             }
-            GroupBox("wallabag Plus is available on my instance ?") {
-                Text("No, wallabag plus is a premium feature only available on your iOS devices")
+            GroupBox("Is wallabag Plus available on my self-hosted instance?") {
```
Comment (bourvill) [inline App/Features/WallabagPlus/WallabagPlusView.swift:12]: Is not only restricted to self-hosted instance. Ex wallabag.it is not self-hosted instance and is not concerned by this feature

**Review by bourvill — COMMENTED** [review]
Thank you very much for your first contribution and this PR. Some elements seem open to discussion; perhaps we could find better terms to explain it more clearly.

File: App/Features/Setting/SettingView.swift (hunk 1 above)
Comment (gabeklavans) [reply App/Features/Setting/SettingView.swift:22]: Ah, my mistake. I thought I tested that but maybe I got confused about the numbers I was seeing. I can re-word.

File: App/Features/WallabagPlus/WallabagPlusView.swift (hunk 2 above)
Comment (gabeklavans) [reply App/Features/WallabagPlus/WallabagPlusView.swift:12]: Yeah I'm not too attached to this change. I just wanted to make it clearer what "my instance" was referring to, but I can just get rid of this line change if it only adds a different type of confusion.

File: App/Features/Setting/SettingView.swift (hunk 1 above)
Comment (gabeklavans) [reply App/Features/Setting/SettingView.swift]: How's the new wording?

**Review by bourvill — APPROVED** [review]
Thank you very much for your contribution!

**Comment by bourvill** [comment]
A new version including your contribution has been sent for validation by Apple. It will be available in version 7.4.0 once accepted by Apple.

**Pull request merged**